src/index.ts

```diff
@@ -71,6 +85,30 @@ export const iconsPlugin = (iconsPluginOptions?: IconsPluginOptions) => {
     })
   }
 
+  const filteredAliases = Object.keys(collectionAliases)
+    .map((key) => ({ key: key, alias: collectionAliases[key] }))
+    .flatMap((alias) => (alias?.alias?.collection ? [alias] : []))
+    .filter((alias) =>
+      isPackageExists(`@iconify-json/${alias.alias!.collection}`),
+    )
```

Comment:
If I only install `@iconify/json`, can I still use `collectionAliases` normally?

Reply:
I'll remove the `filter` package check here and will just rely on `getIconCollections()` package check logic.